components/ep/layout/SurroundDocCard.vue

```diff
@@ -33,6 +33,7 @@ defineProps<{
   <NuxtLink
     :to="withTrailingSlash(doc._path ?? '')"
     class="block max-w-sm p-6 bg-white border border-gray-200 rounded-lg shadow hover:bg-gray-100 dark:bg-gray-800 dark:border-gray-700 dark:hover:bg-gray-700"
+    style="word-wrap: break-word word-break: keep-all"
```

Comment:
You should separate properties in style tag with `;`. I guess these rules are not applied.
```suggestion
    style="word-wrap: break-word; word-break: keep-all"
```

Anyway, inline styles is not a good practice. It is better to use some classes from tailwind (https://tailwindcss.com/docs/word-break) or create class specifically for this card.

Reply:
![image](https://github.com/exactpro/docs-toolkit/assets/62986513/17750237-92f6-4207-a3aa-daf798baa717)
With `;` added it works
Will check if I can use tailwind class as you requested